addons/field_rations/XEH_preInit.sqf

```diff
@@ -13,13 +13,25 @@ GVAR(thirstModifiers) = [];
 GVAR(hungerModifiers) = [];
 
 // List of p3d water sources (case sensitive) for objects that report `typeOf == ""`
-GVAR(waterSourceP3ds) = [
-    "misc_wellpump.p3d" // [11899, 9150, 0] on chenarus
```

Comment:
I would say leave this since it is from a modded map and likely won't have the compat config entry.